packages/kcms/src/team/adaptor/validator/match.ts

```diff
@@ -0,0 +1,38 @@
+import { z } from '@hono/zod-openapi';
+
+const finishStatus = ['goal', 'retired'] as const;
+
+const teamSchema = z.object({
```

Comment:
```suggestion
const TeamSchema = z.object({
```
スキーマは大文字にするルールだったと思います。これ以下も同様です。
また、`team.ts`のほうにあるスキーマと名前が重複してしまうので、`BriefTeamSchema`などと命名したほうが良いかもしれません。